src/config.ts

```diff
@@ -36,55 +37,74 @@ export default class Config {
 
   public static DefaultMessengerType(env: Environment): MessengerType {
     switch (env) {
-      case ENVIRONMENTS.SANDBOX_DEV:
-      case ENVIRONMENTS.ADMIN: // Currently unable to hook into the twitch pubsub system from admin
-      case ENVIRONMENTS.SANDBOX_ADMIN:
+      case Util.Environments.SandboxDev:
+      case Util.Environments.Admin : // Currently unable to hook into the twitch pubsub system from admin
+      case Util.Environments.SandboxAdmin:
         return MessengerType.Pusher;
-      case ENVIRONMENTS.SANDBOX_TWITCH:
-      case ENVIRONMENTS.PRODUCTION:
+      case Util.Environments.SandboxTwitch:
+      case Util.Environments.Production:
         return MessengerType.Twitch;
-      case ENVIRONMENTS.SERVER:
+      case Util.Environments.Server:
         return MessengerType.Server;
+      case Util.Environments.Testing:
     }
     return MessengerType.Unknown;
   }
 
+  public static DefaultPurchaseClientType(env: Environment): PurchaseClientType {
+    switch (env) {
+      case Util.Environments.SandboxDev:
+        return PurchaseClientType.Dev;
+      case Util.Environments.Admin : // Currently unable to hook into the twitch pubsub system from admin
+      case Util.Environments.SandboxAdmin:
+      case Util.Environments.SandboxTwitch:
+      case Util.Environments.Production:
+        return PurchaseClientType.Twitch;
+      case Util.Environments.Server:
+        return PurchaseClientType.Server;
+      case Util.Environments.Testing:
+        return PurchaseClientType.Test;
+    }
+
+    return PurchaseClientType.Unknown;
+  }
+
   public static GetAuthorizationFlowType(env: Environment): AuthorizationFlowType {
     switch (env) {
-      case ENVIRONMENTS.SANDBOX_DEV:
+      case Util.Environments.SandboxDev:
         return AuthorizationFlowType.TestAuth;
-
-      case ENVIRONMENTS.SANDBOX_ADMIN:
-      case ENVIRONMENTS.ADMIN:
+      case Util.Environments.Admin :
         return AuthorizationFlowType.AdminAuth;
-
-      case ENVIRONMENTS.SANDBOX_TWITCH:
-      case ENVIRONMENTS.PRODUCTION:
+      case Util.Environments.SandboxAdmin:
```

Comment:
This looks like you may have moved sandbox admin to the wrong spot. It should end up returning `AdminAuth`